lib/streamlit/hashing.py

```diff
@@ -402,7 +402,7 @@ def _code_to_bytes(self, code, context):
             n
             for n in code.co_consts
             if not isinstance(n, string_types)  # noqa: F821
-            or not n.endswith(".<lambda>")
+                or not n.endswith(".<lambda>")
```

Comment:
Was this indentation change done by [Black](https://black.readthedocs.io/en/stable/) or manually? It looks strange, but if Black did it then it's ok by me

Reply:
I wouldn't have done that, so that was `black`